FlaxEditor/Surface/ContextMenu/VisjectCM.cs

```diff
@@ -111,7 +114,24 @@ private void OnSearchFilterChanged()
 
             // Update groups
             for (int i = 0; i < _groups.Count; i++)
+            {
                 _groups[i].UpdateFilter(_searchBox.Text);
+
+                _groups[i].UpdateItemSort(_selectedBox);
```

Comment:
Adding items score is fine but I think it should be only used when user is connecting boxes (popup shown on mouse button up event) or when user searches for a node. Otherwise scores should be set to 0 to hide the `>` character.

Reply:
I'm glad that you pointed this out. This part of the code had quite a bunch of bugs that I totally missed.